scripts/check_setup.sh

```diff
@@ -33,46 +36,50 @@ else
 fi
 
 # Output required dependencies
-if [ $HAS_REQUIRED = false ] ; then
-    echo "\n\n${COLOR_RED}${BOLD}The following required required dependencies are missing:${RESET_STYLE}\n"
+if [[ $HAS_REQUIRED = false ]] ; then
+    printf "\n\n $COLOR_RED $BOLD The following required required dependencies are missing: $RESET_STYLE \n"
```

Comment:
Looking into it more, didnt know we could use `%b` to evaluate the variables we are substituting